core/src/main/java/com/alibaba/nacos/core/code/StartingSpringApplicationRunListener.java

```diff
@@ -142,8 +141,6 @@ public void failed(ConfigurableApplicationContext context, Throwable exception)
         logFilePath();
         
         LOGGER.error("Startup errors : {}", exception);
-        
-        HttpClientManager.shutdown();
```

Comment:
HttpClientBeanHolder should be shutdown in here?

Reply:
Will submit a new pr to solve this problem, link https://github.com/alibaba/nacos/issues/3832